src/script/lua_api/l_util.cpp

```diff
@@ -477,6 +478,23 @@ int ModApiUtil::l_sha1(lua_State *L)
 	return 1;
 }
 
+// colorspec_to_colorstring(colorspec)
+int ModApiUtil::l_colorspec_to_colorstring(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+
+	video::SColor color(0);
+	if (read_color(L, 1, &color)) {
+		char colorstring[10];
+		sprintf(colorstring, "#%02X%02X%02X%02X",
```

Comment:
Should be `snprintf`

Reply:
It's not possible for that format string to overflow the buffer length, but I suppose it can't hurt to be careful.